Source/Core/RequestScheduler.js

```diff
@@ -182,6 +182,8 @@ function getRequestReceivedFunction(request) {
     requestCompletedEvent.raiseEvent();
     request.state = RequestState.RECEIVED;
     request.deferred.resolve(results);
+    // explicitly set to undefined to ensure GC of request response data. See #8843
+    request.deferred = undefined;
```

Comment:
@lilleyse I have a regression introduced by this change, but it may be an unrelated bug that this is just covering up, not sure.  Either way it's a RequestScheduler issue.  I'm trying to create a simple reproduction now and then I'll write it up.  Hopefully it's an easy fix (since you know this code way better than I do).